core/src/main/java/com/linecorp/armeria/server/annotation/ResponseConverterFunction.java

```diff
@@ -36,6 +36,19 @@
      * Returns {@link HttpResponse} instance corresponds to the given {@code result}.
      * Calls {@link ResponseConverterFunction#fallthrough()} or throws a {@link FallthroughException} if
      * this converter cannot convert the {@code result} to the {@link HttpResponse}.
+     *
+     * @param headers The HTTP headers of the {@link HttpResult} returned by the service method.
+     *                Even if the service method does not return any {@link HttpResult},
+     *                the headers will basically include {@code "status"} and {@code "content-type"}
+     *                HTTP Response Header.
+     *                Note that the additional headers will not be included in this headers,
+     *                call {@link ServiceRequestContext#additionalResponseHeaders()} to access them.
```

Comment:
I think we should explain the default status is `200` unless a user specifies it with `@StatusCode`.
I'm not sure that this is best describable, but how about this?

```
@param headers the basic HTTP headers that you might want to use to create the {@link HttpResponse} with.
     *                The status of headers is {@link HttpStatus#OK} by default, unless you specify it with
     *                {@link StatusCode} on the method. The headers also will include a {@link MediaType}, if
     *                {@link ServiceRequestContext#negotiatedResponseMediaType()} returns it.
     *                If the method that you decorate with this function returns an {@link HttpResult},
     *                this headers is the same header from {@link HttpResult#headers()}.
     *                The headers could be immutable, so please call {@link HttpHeaders#toMutable()} before you modify it.
     *                Please note that the additional headers which are set by {@link ServiceRequestContext#addAdditionalResponseHeader(AsciiString, String)}
     *                and {@link AdditionalHeaders} are not included in this header.
```

Reply:
@minwoox 👍 
How about this?
- If the method that you decorate with this function returns -> If the annotated method returns